### PR TITLE
Vickydev

### PR DESCRIPTION
**Pull request by nag243**

### CODE REVIEW

File: src/app/app-routing.module.ts [hunk 1]
```diff
@@ -28,9 +28,17 @@ const routes: Routes = [
     path: 'my-account/forgot-password',
     loadChildren: () => import('./views/auth-views/forgot-password/forgot-password.module').then(m => m.ForgotPasswordModule)
   },
+  {
+    path: 'admin/category',
+    loadChildren: () => import('./views/admin-views/category/category.module').then(m => m.CategoryModule)
+  },
+  {
+    path: 'admin/listof-category',
```
Comment (Nithyam1330) [inline src/app/app-routing.module.ts]: It should be admin/category-list

File: src/app/app-routing.module.ts [hunk 2]
```diff
@@ -28,9 +28,17 @@ const routes: Routes = [
     path: 'my-account/forgot-password',
     loadChildren: () => import('./views/auth-views/forgot-password/forgot-password.module').then(m => m.ForgotPasswordModule)
   },
+  {
+    path: 'admin/category',
```
Comment (Nithyam1330) [inline src/app/app-routing.module.ts]: Remove Category

File: src/app/app.module.ts [hunk 3]
```diff
@@ -16,7 +17,8 @@ import { AdminProductsListComponent } from './views/admin-views/admin-products-l
   imports: [
     BrowserModule,
     AppRoutingModule,
-    SharedModule
+    SharedModule,
+    NgbModule,NgbAlertModule,
```
Comment (Nithyam1330) [inline src/app/app.module.ts]: Remove this as we have already in shared module

File: src/app/app.module.ts [hunk 4]
```diff
@@ -6,6 +6,7 @@ import { AppComponent } from './app.component';
 import { SharedModule } from './shared/shared/shared.module';
 import { TopbarComponent } from './shared/shared/components/topbar/topbar.component';
 import { AdminProductsListComponent } from './views/admin-views/admin-products-list/admin-products-list.component';
+import {NgbModule,NgbAlertModule} from '@ng-bootstrap/ng-bootstrap';
```
Comment (Nithyam1330) [inline src/app/app.module.ts]: Remove this import

File: src/app/views/admin-views/category/category-routing.module.ts [hunk 5]
```diff
@@ -1,8 +1,14 @@
 import { NgModule } from '@angular/core';
```
Comment (Nithyam1330) [inline src/app/views/admin-views/category/category-routing.module.ts]: Remove this file as we dont have route

File: src/app/views/admin-views/listof-category/listof-category-routing.module.ts [hunk 6]
```diff
@@ -0,0 +1,17 @@
+import { NgModule } from '@angular/core';
+import { Routes, RouterModule } from '@angular/router';
+import { ListofCategoryComponent } from './listof-category.component';
+
+
+const routes: Routes = [
+  {
+    path:'',
+    component:ListofCategoryComponent
```
Comment (Nithyam1330) [inline src/app/views/admin-views/listof-category/listof-category-routing.module.ts]: Change component name to categoryListComponent

File: src/app/views/admin-views/listof-category/listof-category-routing.module.ts [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+  }
+];
+
+@NgModule({
+  imports: [RouterModule.forChild(routes)],
+  exports: [RouterModule]
+})
+export class ListofCategoryRoutingModule { }
```
Comment (Nithyam1330) [inline src/app/views/admin-views/listof-category/listof-category-routing.module.ts]: Change module to CategoryListRoutingModule

File: src/app/views/admin-views/listof-category/listof-category.component.html [hunk 8]
```diff
@@ -0,0 +1,29 @@
+<div class="d-flex flex-wrap  justify-content-center align-items-center">
+    <div class="card m-t-100" style="width: 70rem;">
+        <div class="card-body">
+            <div class="d-flex justify-content-end">
+                <button class="btn btn-primary" [routerLink]="['../category']">Add Catagory</button>
+            </div>
+            <div>
+                <table class="table table-striped">
```
Comment (Nithyam1330) [inline src/app/views/admin-views/listof-category/listof-category.component.html]: Use table same as product list

File: src/app/views/admin-views/listof-category/listof-category.module.ts [hunk 9]
```diff
@@ -0,0 +1,15 @@
...
+
+
+@NgModule({
+  declarations: [ListofCategoryComponent],
+  imports: [
+    CommonModule,
+    ListofCategoryRoutingModule
+  ]
+})
+export class ListofCategoryModule { }
```
Comment (Nithyam1330) [inline src/app/views/admin-views/listof-category/listof-category.module.ts]: It should be CategoryListModule

File: src/app/views/auth-views/forgot-password/forgot-password.component.html [hunk 10]
```diff
@@ -1 +1,29 @@
...
+                <div class="form-group">
+                    <label for="exampleInputEmail1">Email </label>
+                    <input type="email" class="form-control" id="exampleInputEmail1" aria-describedby="emailHelp">
+                </div>
+                <div class="d-flex justify-content-between">
+                    <div class="d-flex justify-content-start">
+                        <a [routerLink]="['../login']" class="btn btn-link ">Login</a>
+                    </div>
+                    <div class="d-flex justify-content-start">
+                        <a [routerLink]="['../registration']" class="btn btn-link ">Registration</a>
```
Comment (Nithyam1330) [inline src/app/views/auth-views/forgot-password/forgot-password.component.html]: Will not navigate

File: src/app/views/auth-views/login/login.component.html [hunk 11]
```diff
@@ -1,10 +1,34 @@
...
+<div class="d-flex flex-wrap  justify-content-center align-items-center">
+    <div class="card m-t-50" style="width: 22rem;">
+        <div class="card-body">
+            <div class="d-flex flex-wrap  justify-content-center align-items-center">
+                <p style="font-size: medium;">Login</p>
+            </div>
+            <form>
+                <div class="form-group">
+                    <label for="exampleInputEmail1">Email </label>
+                    <input type="email" class="form-control" id="exampleInputEmail1" aria-describedby="emailHelp">
```
Comment (Nithyam1330) [inline src/app/views/auth-views/login/login.component.html]: Remove aria-describedby attribute and chnage the ID value

File: src/app/views/auth-views/login/login.component.html [hunk 12]
```diff
@@ -1,10 +1,34 @@
...
+                </div>
+                <div class="form-group">
+                    <label for="exampleInputPassword1">Password</label>
+                    <input type="password" class="form-control" id="exampleInputPassword1">
+                </div>
+                <div class="d-flex justify-content-between">
+                    <div class="d-flex justify-content-start">
+                        <a [routerLink]="['../registration']" class="btn btn-link ">Registration</a>
```
Comment (Nithyam1330) [inline src/app/views/auth-views/login/login.component.html]: Will not work please change router link value

File: src/app/views/auth-views/login/login.component.html [hunk 13]
```diff
@@ -1,10 +1,34 @@
...
+                    </div>
+                    <div class="d-flex justify-content-end">
+                        <a [routerLink]="['../forgot-password']" class="btn btn-link">Forgot Password?</a>
```
Comment (Nithyam1330) [inline src/app/views/auth-views/login/login.component.html]: Will not work please change router link value

File: src/app/views/auth-views/registration/registration.component.html [hunk 14]
```diff
@@ -1 +1,42 @@
...
+                <div>
+                    <label>
+                    <input type="checkbox"> I agree with terms & conditions</label>
+                </div>
+                <div class="d-flex justify-content-between">
+                    <div class="d-flex justify-content-start">
+                        <a [routerLink]="['../login']" class="btn btn-link ">Login</a>
+                    </div>
+                    <div class="d-flex justify-content-end">
+                        <a [routerLink]="['../forgot-password']" class="btn btn-link">Forgot Password?</a>
```
Comment (Nithyam1330) [inline src/app/views/auth-views/registration/registration.component.html]: Will not work please change router link value

File: src/app/views/auth-views/registration/registration.component.html [hunk 15]
```diff
@@ -1 +1,42 @@
...
+                    <label for="exampleInputPassword1">Phone Number</label>
+                    <input type="number" class="form-control">
+                </div>
+                <div>
+                    <label>
+                    <input type="checkbox"> I agree with terms & conditions</label>
+                </div>
+                <div class="d-flex justify-content-between">
+                    <div class="d-flex justify-content-start">
+                        <a [routerLink]="['../login']" class="btn btn-link ">Login</a>
```
Comment (Nithyam1330) [inline src/app/views/auth-views/registration/registration.component.html]: Will not work please change router link value

File: src/app/views/auth-views/registration/registration.component.html [hunk 16]
```diff
@@ -1 +1,42 @@
-<p>registration works!</p>
+<div class="d-flex flex-wrap  justify-content-center align-items-center">
+    <div class="card m-t-50" style="width: 22rem;">
+        <div class="card-body">
+            <div class="d-flex flex-wrap  justify-content-center align-items-center">
+                <p style="font-size: medium;">Registration</p>
```
Comment (Nithyam1330) [inline src/app/views/auth-views/registration/registration.component.html]: Dont use inline styles. Instead use a CSS class

File: src/app/views/auth-views/registration/registration.component.html [hunk 17]
```diff
@@ -1 +1,42 @@
...
+            </div>
+            <form>
+                <div class="form-group">
+                    <label for="exampleInputPassword1">Full Name</label>
+                    <input type="password" class="form-control">
```
Comment (Nithyam1330) [inline src/app/views/auth-views/registration/registration.component.html]: Type should be text

File: src/app/views/auth-views/registration/registration.component.html [hunk 18]
```diff
@@ -1 +1,42 @@
...
+                </div>
+                <div class="form-group">
+                    <label for="exampleInputEmail1">Email </label>
+                    <input type="email" class="form-control" aria-describedby="emailHelp">
```
Comment (Nithyam1330) [inline src/app/views/auth-views/registration/registration.component.html]: Remove aria-describedby attribute and change for attribute in label tag

File: src/app/views/auth-views/registration/registration.component.html [hunk 19]
```diff
@@ -1 +1,42 @@
...
+                </div>
+                <div class="form-group">
+                    <label for="exampleInputPassword1">Phone Number</label>
```
Comment (Nithyam1330) [inline src/app/views/auth-views/registration/registration.component.html]: change for attribute value

File: src/app/views/auth-views/registration/registration.component.html (hunk 15 above)
Comment (Nithyam1330) [inline src/app/views/auth-views/registration/registration.component.html]: Will not work please change router link value

File: src/app/views/auth-views/registration/registration.component.html (hunk 14 above)
Comment (Nithyam1330) [inline src/app/views/auth-views/registration/registration.component.html]: Will not work please change router link value

File: src/app/views/user-views/dashboard/dashboard.component.html [hunk 20]
```diff
@@ -0,0 +1,12 @@
+<ngb-carousel>
+    <ng-container *ngFor="let img of images ">
```
Comment (Nithyam1330) [inline src/app/views/user-views/dashboard/dashboard.component.html]: Use image name instead of img

Comment (nag243) [reply src/app/views/user-views/dashboard/dashboard.component.html]: image replaced.

File: src/app/views/auth-views/registration/registration.component.html (hunk 14 above)
Comment (nag243) [reply src/app/views/auth-views/registration/registration.component.html]: resolved.

File: src/app/views/auth-views/registration/registration.component.html (hunk 15 above)
Comment (nag243) [reply src/app/views/auth-views/registration/registration.component.html]: Resolved.

File: src/app/views/auth-views/registration/registration.component.html (hunk 19 above)
Comment (nag243) [reply src/app/views/auth-views/registration/registration.component.html]: Resolved.

File: src/app/views/auth-views/registration/registration.component.html (hunk 18 above)
Comment (nag243) [reply src/app/views/auth-views/registration/registration.component.html]: Resolved.

File: src/app/views/auth-views/registration/registration.component.html (hunk 17 above)
Comment (nag243) [reply src/app/views/auth-views/registration/registration.component.html]: Resolved.

Comment (nag243) [reply src/app/views/auth-views/registration/registration.component.html]: Resolved.

File: src/app/views/auth-views/registration/registration.component.html (hunk 16 above)
Comment (nag243) [reply src/app/views/auth-views/registration/registration.component.html]: Resolved.

File: src/app/views/auth-views/registration/registration.component.html (hunk 15 above)
Comment (nag243) [reply src/app/views/auth-views/registration/registration.component.html]: Resolved.

File: src/app/views/auth-views/registration/registration.component.html (hunk 14 above)
Comment (nag243) [reply src/app/views/auth-views/registration/registration.component.html]: Resolved.

File: src/app/views/auth-views/login/login.component.html (hunk 13 above)
Comment (nag243) [reply src/app/views/auth-views/login/login.component.html]: Reslioved.

File: src/app/views/auth-views/login/login.component.html (hunk 12 above)
Comment (nag243) [reply src/app/views/auth-views/login/login.component.html]: Resolved.

File: src/app/views/auth-views/login/login.component.html (hunk 11 above)
Comment (nag243) [reply src/app/views/auth-views/login/login.component.html]: Resolved.

File: src/app/views/auth-views/forgot-password/forgot-password.component.html (hunk 10 above)
Comment (nag243) [reply src/app/views/auth-views/forgot-password/forgot-password.component.html]: Resolved.

File: src/app/app-routing.module.ts (hunk 1 above)
Comment (nag243) [reply src/app/app-routing.module.ts]: Resolved.

File: src/app/app-routing.module.ts (hunk 2 above)
Comment (nag243) [reply src/app/app-routing.module.ts]: Resolved.

File: src/app/app.module.ts (hunk 3 above)
Comment (nag243) [reply src/app/app.module.ts]: Resolved.

File: src/app/app.module.ts (hunk 4 above)
Comment (nag243) [reply src/app/app.module.ts]: Resolved.

File: src/app/views/admin-views/listof-category/listof-category.module.ts (hunk 9 above)
Comment (nag243) [reply src/app/views/admin-views/listof-category/listof-category.module.ts]: Resolved.

File: src/app/views/admin-views/listof-category/listof-category-routing.module.ts (hunk 6 above)
Comment (nag243) [reply src/app/views/admin-views/listof-category/listof-category-routing.module.ts]: Resolved.

File: src/app/views/admin-views/listof-category/listof-category-routing.module.ts (hunk 7 above)
Comment (nag243) [reply src/app/views/admin-views/listof-category/listof-category-routing.module.ts]: Resolved.

File: src/app/app-routing.module.ts (hunk 1 above)
Comment (Nithyam1330) [reply src/app/app-routing.module.ts]: change the URL ro admin/category-list

File: src/app/app.module.ts (hunk 4 above)
Comment (Nithyam1330) [reply src/app/app.module.ts]: Not resolved please check again

File: src/app/app.module.ts (hunk 3 above)
Comment (Nithyam1330) [reply src/app/app.module.ts]: Not resolved please check again

File: src/app/views/admin-views/listof-category/listof-category-routing.module.ts (hunk 6 above)
Comment (Nithyam1330) [reply src/app/views/admin-views/listof-category/listof-category-routing.module.ts]: Not resolved please check

Comment (nag243) [reply src/app/views/admin-views/listof-category/listof-category-routing.module.ts]: Resolved.

File: src/app/app.module.ts (hunk 4 above)
Comment (nag243) [reply src/app/app.module.ts]: Resolved.

File: src/app/app.module.ts (hunk 3 above)
Comment (nag243) [reply src/app/app.module.ts]: Resolved.

File: src/app/app-routing.module.ts (hunk 1 above)
Comment (nag243) [reply src/app/app-routing.module.ts]: Resolved.

**Pull request merged**